recipes/salt/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+SRC_DIR="."
+
+$PYTHON setup.py install
+
+# Create directory structure for salt
+# These are ENV safe, because we created a custom
+# _syspaths.py file
+
+# this file re-maps the default paths for configs
+cp $RECIPE_DIR/_syspaths.py $SP_DIR/salt
+
+DIRECTORIES="etc/salt
+var/cache/salt
+var/run/salt
+srv/salt
+srv/pillar
+var/log/salt
+var/run
+"
+for path in $DIRECTORIES
+do
+    mkdir -p $PREFIX/$path
+    touch $PREFIX/$path/.gitkeep
```

Comment:
Salt uses that structure of directories to store some information (keys and configs) i think some of these are optional but i found it better to create them in the recipe.